app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,24 +58,26 @@ module SupportsFeatureMixin
   extend ActiveSupport::Concern
 
   QUERYABLE_FEATURES = {
-    :associate_floating_ip    => 'Associate a Floating IP',
-    :control                  => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
-    :cloud_tenant_mapping     => 'CloudTenant mapping',
-    :cinder_service           => 'Cinder storage service',
-    :delete                   => 'Deletion',
-    :disassociate_floating_ip => 'Disassociate a Floating IP',
-    :discovery                => 'Discovery of Managers for a Provider',
-    :evacuate                 => 'Evacuation',
-    :events                   => 'Query for events',
-    :launch_cockpit           => 'Launch Cockpit UI',
-    :live_migrate             => 'Live Migration',
-    :migrate                  => 'Migration',
-    :provisioning             => 'Provisioning',
-    :reconfigure              => 'Reconfiguration',
-    :regions                  => 'Regions of a Provider',
-    :resize                   => 'Resizing',
-    :retire                   => 'Retirement',
-    :smartstate_analysis      => 'Smartstate Analaysis',
+    :associate_floating_ip        => 'Associate a Floating IP',
+    :control                      => 'Basic control operations', # FIXME: this is just a internal helper and should be refactored
+    :cloud_tenant_mapping         => 'CloudTenant mapping',
+    :cloud_volume_backup_create   => 'CloudVolume backup creation',
+    :cloud_volume_backup_restore  => 'CloudVolume backup restore',
+    :cinder_service               => 'Cinder storage service',
```

Comment:
@Fryguy @durandom My understanding was that this was used to support the pattern of adding an action that might only be supported by one provider, but which may be supported by additional providers in the future.  See the comments on https://github.com/ManageIQ/manageiq/pull/9475 for an example of where we were asked to go this way.